test/Sentry.Tests/Sentry.Tests.csproj

```diff
@@ -43,4 +43,10 @@
     <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="8.0.1" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' == 'net9.0' OR '$(TargetFramework)' == 'net8.0'">
```

Comment:
question: Should we create a separate `Sentry.SourceGenerators.Tests` project?

To be consistent with the rest of our `src`-`test`-Project-pairs?

Reply:
I pushed it into Sentry.Tests because it is technically bundled as part of Sentry.  There were also various references I need already found within Sentry.Tests